src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/CCDCaseApi.java

```diff
@@ -172,12 +173,18 @@ private void linkToCaseWithoutRevoking(
     }
 
     private List<Claim> getAllCasesBy(User user, ImmutableMap<String, String> searchString) {
-        List<CaseDetails> validCases = searchAll(user, searchString)
-            .stream()
-            .filter(caseDetails -> !isCreatedState(caseDetails))
+        return extractClaims(searchAll(user, searchString));
+    }
+
+    private List<Claim> getAllIssuedCasesBy(User user, ImmutableMap<String, String> searchString) {
+        List<CaseDetails> issuedCases = asStream(searchAll(user, searchString))
+            .filter(isCaseIssued())
             .collect(Collectors.toList());
+        return extractClaims(issuedCases);
+    }
 
-        return extractClaims(validCases);
+    private Predicate<CaseDetails> isCaseIssued() {
+        return caseDetails -> !CREATE.getValue().equals(caseDetails.getState());
```

Comment:
If you intend to get only open case I would suggest you to use 
`caseDetails -> !OPEN.getValue().equals(caseDetails.getState());` instead because `!CREATE` can mean closed and settled also

Reply:
that's exactly what we are looking here anything other than CREATE

Reply:
same as above